Gems/Multiplayer/Code/Include/Multiplayer/MultiplayerEditorConnectionViewportDebugBus.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+
+namespace Multiplayer
+{
+    //! Request bus for displaying useful debug information in the editor viewport while the editor is making its connection to the editor-server.
+    //! Note: This isn't used once the Editor has connected to the Multiplayer simulation itself.
+    class MultiplayerEditorConnectionViewportDebugRequests : public AZ::EBusTraits
```

Comment:
Might as well be an AZ::Interface, no? it would make all your calling code more straight-forward.

Reply:
Fixed!